synapse/config/server.py

```diff
@@ -1104,11 +1121,22 @@ def generate_config_section(
           #  type: manhole
 
         # Connection settings for the manhole
+        #
         manhole_settings:
-        #  username: matrix
-        #  password: rabbithole
-        #  ssh_priv_key: CONFDIR/id_rsa
-        #  ssh_pub_key: CONFDIR/id_rsa.pub
+
+          # The username for the manhole. This defaults to matrix
+          #
+          # username: matrix
+
+          # The password for the manhole. This defaults to rabbithole
+          #
+          # password: rabbithole
+
+          # The private and public SSH key pair used to encrypt the manhole traffic.
+          # If these are left unset, then hardcoded and non-secret keys are used!
```

Comment:
this is only a problem if the connection goes over the public internet? We should probably say that.